src/Views/Appearance.vala

```diff
@@ -110,6 +110,34 @@ public class PantheonShell.Appearance : Gtk.Grid {
         };
         dark_info.get_style_context ().add_class (Gtk.STYLE_CLASS_DIM_LABEL);
 
+        var schedule_label = new Gtk.Label (_("Schedule:")) {
+            halign = Gtk.Align.END,
+            xalign = 1
+        };
+
+        var schedule_mode_button = new Granite.Widgets.ModeButton ();
+        schedule_mode_button.append_text (_("Disabled"));
+        schedule_mode_button.append_text (_("Sunset to Sunrise"));
+        schedule_mode_button.append_text (_("Manual"));
+
+        var from_label = new Gtk.Label (_("From:"));
+
+        var from_time = new Granite.Widgets.TimePicker ();
+
+        var to_label = new Gtk.Label (_("To:"));
+
+        var to_time = new Granite.Widgets.TimePicker ();
+
+        var schedule_grid = new Gtk.Grid () {
+            column_spacing = 12,
+            row_spacing = 12
+        };
+
+        schedule_grid.attach (from_label, 0, 0, 1, 1);
+        schedule_grid.attach (from_time, 1, 0, 1, 1);
+        schedule_grid.attach (to_label, 2, 0, 1, 1);
+        schedule_grid.attach (to_time, 3, 0, 1, 1);
```

Comment:
Since these are just horizontal in a row it's easier to do

```suggestion
        schedule_grid.add (from_label);
        schedule_grid.add (from_time);
        schedule_grid.add (to_label);
        schedule_grid.add (to_time);
```